### DISCUSSION TITLE
Daily 09/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#564] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé con la tarea de conectar el Perfil al backend, gracias a la ayuda de Roberto. Ya está subido al repo y funciona correctamente.
Hoy voy a realizar las tareas de frontend que tengo asignadas. También trataré de revisar la documentación para ya tenerla lista.

**Comment by rpgrca** [comment]
Hoy tuve clase de inglés. En cuanto al proyecto separé el models.py en varios archivos, uno por cada modelo en un directorio models, y también separé los serializadores de serializers.py en un directorio serializers para que podamos trabajar cada uno en uno sin que hayan tantos conflictos. Voy a armar un formulario simple para asignar artículos a oferta y terminar lo de la oferta. Mañana no voy a estar disponible durante el día así que seguiré el domingo (sí, los fines se trabaja :1st_place_medal: )

**Comment by BraianTroncoso** [comment]
Hoy tuve clase de Inglés. En cuanto al proyecto voy a seguir trabajando en las issues asignadas estuve con mucho trabajo y no le dediqué el tiempo que quisiera, este finde lo tengo libre para poder estudiar, voy a enfocarme asi puedo cumplir con el objetivo planteado.
